cmd/cluster/create_cluster.go

```diff
@@ -128,11 +129,30 @@ var createClusterCmd = &cobra.Command{
 		createClusterRequest := ybmclient.NewCreateClusterRequest(*clusterSpec)
 		createClusterRequest.SetEncryptedDbCredentials(*dbCredentials)
 
+		// Enable connection pooling feature if requested
+		enableConnectionPooling, _ := cmd.Flags().GetBool("enable-connection-pooling")
+		if enableConnectionPooling {
+			if !util.IsFeatureFlagEnabled(util.CONNECTION_POOLING) {
+				logrus.Fatalf("Connection pooling feature is not enabled. Please set YBM_FF_CONNECTION_POOLING=true environment variable")
```

Comment:
I don't think we need to inform the customers about the feature flag. We need to remove it/make it true on our side when it is ready for the public.